tools/myst/enc/enc.c

```diff
@@ -496,8 +496,8 @@ static long _enter(void* arg_)
     myst_args_t env;
     const char* cwd = "/";       // default to root dir
     const char* hostname = NULL; // kernel has a default
-    const uint8_t* enclave_base;
-    size_t enclave_size;
+    const uint8_t* enclave_image_base;
```

Comment:
Does it matter to call the variable `enclave_image_base` instead of `enclave_base`?

Reply:
This is to avoid the confusion: when enabling the 0-base feature, `__oe_get_enclave_base_address()` (previously called `__oe_get_enclave_base`) will return 0. Also in case that we may add a check using `__oe_get_enclave_base_address()` in the future.